### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jgoizueta**
See #4

### CODE REVIEW

**Comment by jgoizueta** [comment]
@rafatower please review

File: README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+and further developed by CARTO <dataservices@carto.com> since 2016.
+
+Building and Installing
+-----------------------
+
+The extension can be built and installed with:
+
+```sh
+PATH=$(pg_config --bindir):$PATH make
+sudo PATH=$(pg_config --bindir):$PATH make install
```
Comment (rafatower) [inline README.md]: I don't agree with the PATH tweaks. Is there a reason to do so?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+ODBC FDW for PostgreSQL 9.5+
+============================
+
+This PostgreSQL extension implements a Foreign Data Wrapper (FDW) for
+remote databases using Open Database Connectivity [ODBC](http://msdn.microsoft.com/en-us/library/ms714562(v=VS.85).aspx).
+
+This was originally developed by Zheng Yang <zhengyang4k@gmail.com> in 2011,
+with contributions by Gunnar "Nick" Bluth <nick@pro-open.de> from 2014
+and further developed by CARTO <dataservices@carto.com> since 2016.
+
```
Comment (rafatower) [inline README.md]: Missing a requirements section

File: README.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+the ODBC attributes to define a connection to an ODBC data source
+and some additional parameters to specify the table or query that
+will be accessed as a foreign table.
+
+The following options to define ODBC attributes should be defined in
+the server definition (`CREATE SERVER`).
+
+option   | description
+-------- | -----------
+`dsn`    | The Database Source Name for the foreign database system you're connecting to.
```
Comment (rafatower) [inline README.md]: `s/for/of`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+`driver` | The name of the ODBC driver to use (needed if no dsn is used)
+
+These additional ODBC connection options are supported and can be defined
+either in the server or foreign table definition (or in an IMPORT FOREIGN SCHEMA statement):
+
+option     | description
+---------- | -----------
+`host`     | The name of the server which provides the database.
```
Comment (rafatower) [inline README.md]: I'd simply say "the hostname or ip of the database server", or am I wrong maybe?

does it support a plain ip?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+`port`     | The server port to connect to.
+`database` | The name of the database to query.
+`username` | The username to authenticate to the foreign server with.
+`password` | The password to authenticate to the foreign server with.
```
Comment (rafatower) [inline README.md]: "to authenticate in the foreign server"

File: README.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+`sql_query`| Optional: User defined SQL statement for querying the foreign table(s). This overrides the `table` parameters. This should use the syntax of ODBC driver used.
+`sql_count`| Optional: User defined SQL statement for counting number of records in the foreign table(s). This should use the syntax of ODBC driver used.
+`prefix`   | For IMPORT FOREIGN SCHEMA: a prefix for foreign table names. This can be used to prepend a prefix to the names of tables imported from an external database.
+
+Example
+-------
+
+Assuming that the `odbc_fdw` is installed and available
+in our database (`CREATE EXTENSION odbc_fdw`), and that
+we have a DNS `test` defined for some ODBC datasource which
```
Comment (rafatower) [inline README.md]: Typo plus style improvements: "in *your* database... that *you* have a **DSN**"

**Comment by rafatower** [comment]
Overall OK. I just left some minor suggestions.

File: README.md (hunk 1 above)
Comment (jgoizueta) [inline README.md]: Just copied from the old README. Shouldn't be needed, I'll remove it.

File: README.md (hunk 2 above)
Comment (jgoizueta) [inline README.md]: I'm adding it.

File: README.md (hunk 4 above)
Comment (jgoizueta) [inline README.md]: You'r right, its the server address

**Comment by jgoizueta** [comment]
@rafatower: I've done the corrections, can you look at the new requirements section?

**Comment by rafatower** [comment]
:+1:

**Pull request merged**